.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
```

Comment:
It doesn't bother me, so only mentioning in case it does you, but the job name here is `Build`, whereas that for the sibling SDK repo is `Test`, even though the workflows themselves are very similar.

https://github.com/forcedotcom/sf-fx-sdk-nodejs/pull/147/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R1

Reply:
i'll comment on the other PR for this